app/src/main/java/chat/rocket/android/main/presentation/MainNavigator.kt

```diff
@@ -48,6 +49,12 @@ class MainNavigator(internal val activity: MainActivity) {
         }
     }
 
+    fun toProfileImageDialog(avatarUrl: String) {
```

Comment:
`toProfileImage` is enough - the "same" way we have declaring other functions here.